app/services/importers/ok_sos/split_files.rb

```diff
@@ -38,6 +38,8 @@ def perform
       end
 
       def import_files
+        ::Importers::OkSos::ZipCodes.perform(@csvs["entity_addresses"].path)
```

Comment:
Loading zip codes from entity addresses saves some time on the first run. Thousands of zips are loaded from all over the country so there's quite a few.